CONTRIBUTING.md

```diff
@@ -88,17 +90,18 @@ When a new Expo SDK is released, a new matching `@bugsnag/expo` version needs to
 
 To start a release:
```

Comment:
do these update instructions look right @gingerbenw ?

Reply:
yeah, seems good to me - should we add branch protection on the next vX and vX-next branches in github?

Reply:
i.e. and add that to the contributing guide?